2 - RTB/Documentazione esterna/Analisi dei Requisiti/Analisi dei Requisiti.typ

```diff
@@ -539,3 +539,60 @@ $bold("Scenario: ")$
 - l'utente seleziona un bin;
 - vengono visualizzate le informazioni sul bin e, se presente, sul prodotto contenuto nel bin.
 
+= Esplorazione magazzino
+
+#figure(image("./imgs/uc7.png", format: "png"), caption: [UML UC-7])
+
+== Spostamento della visuale
+$bold("Descrizione: ")$
+Una volta che il sistema è stato configurato l'utente può visualizzare il magazzino e spostare la visuale sui quattro assi (avani, indietro, destra e sinistra).
```

Comment:
In realtà il movimento è formalmente corretto definirlo su 3 assi (x,y,z)... 4 sono le direzioni. Cambierei in "sui 3 assi" evitando la parte tra parentesi.